roles/cinder-common/templates/etc/cinder/cinder.conf

```diff
@@ -45,16 +38,24 @@ backup_driver = cinder.backup.drivers.swift
 backup_swift_url = http://{{ endpoints.swift.haproxy_vip }}:{{ endpoints.swift.port.cinder_backup }}/v1/
 {% endif -%}
 
+{% if not ceph.enabled|bool -%}
+default_volume_type = LVM
+enabled_backends = lvm
+[lvm]
+volume_backend_name = lvm
+volume_driver = cinder.volume.drivers.lvm.LVMVolumeDriver
+iscsi_helper=tgtadm
+iscsi_ip_address = {{ primary_ip }}
+volume_group = cinder-volumes
+use_forwarded_for = true
+{% endif -%}
+
 {% if ceph.enabled|bool -%}
 {% if cinder.common_volume_name -%}
 host = ceph
 {% endif -%}
 
-{% if cinder.default_volume_type | default(false) -%}
```

Comment:
Any changes to confs etc should be well thought as it can cause backward compatibility issues / breakages in productions.